src/EnergyPlus/DXCoils.cc

```diff
@@ -8456,7 +8458,7 @@ Label50: ;
 			}
 			EIR = DXCoil( DXCoilNum ).RatedEIR( Mode ) * EIRTempModFac * EIRFlowModFac;
 			// Calculate modified PartLoadRatio due to defrost (reverse-cycle defrost only)
-			PLRHeating = min( 1.0, ( PartLoadRatio + LoadDueToDefrost / TotCap ) );
+			PLRHeating = min( 1.0, ( PartLoadRatio + LoadDueToDefrost / TotCapAdj ) );
 			if ( DXCoil( DXCoilNum ).DXCoilType_Num != CoilVRF_Heating ) {
```

Comment:
This one is tricky. What do we add to PLR? It's the heating load missed (assumed calculated correctly) to the available heating capacity. Here PLR is the ratio of load to available heating capacity wo defrost. So do we add load missed divided by available or adjusted heating capacity? I think it's the former, but your guess is as good as mine here. How do I resolve this one?